matter/efr32/efr32mg12/BRD4162A/autogen/sl_event_handler.h

```diff
@@ -7,5 +7,6 @@ void sl_driver_init(void);
 void sl_service_init(void);
 void sl_stack_init(void);
 void sl_internal_app_init(void);
+void sl_iostream_init_instances(void);
```

Comment:
Why this one here ? isn't `sl_iostream_init_instances` proto defined in `sl_iostream_init_instances.h`

Reply:
I don't know why SLC adds the `sl_iostream_init_instances()` definition to `sl_event_handler.h`, but it is redundant in the context of `sl_event_handler.c`.
I have removed the line from `sl_event_handler.h`.